### PR TITLE
Fix/PC Console - Allow functions for search query

### PR DESCRIPTION
**Pull request by xkopenreview**
this pr should add support for function strings so that the query search can search for arbitrary field

### CODE REVIEW

File: components/webfield/ProgramChairConsole/PaperStatusMenuBar.js [hunk 1]
```diff
@@ -72,9 +72,46 @@ const PaperStatusMenuBar = ({
         }),
         {}
       )),
-    ...(typeof extraPropertiesAllowed === 'object' && extraPropertiesAllowed),
+    ...(typeof extraPropertiesAllowed === 'object' &&
+      Object.fromEntries(
+        Object.entries(extraPropertiesAllowed).map(([key, value]) => {
+          if (typeof value === 'string') {
+            return [key, [key]]
```
Comment (zbialecki) [inline components/webfield/ProgramChairConsole/PaperStatusMenuBar.js:79]: Can you explain what this is doing? Why is it discarding value if it's a string?

**Review by zbialecki — COMMENTED** [review]
My issue with this PR is that this pattern you're using of `Object.fromEntries(Object.entries(...).map(...))` isn't efficient (because of the multiple iterations and allocations) and it's harder to read than having a simple forEach or using _.mapValues. It's especially important to optimize this when you're iterating over all rows, like you are on line 104.

**Comment by melisabok** [comment]
@xkopenreview can you address the comments here so we can use this PR for the next ARR June cycle?

**Comment by melisabok** [comment]
ohhh I see the commit. 

Could you put an example of the configuration so I can test it using a test in openreview-py?

**Comment by melisabok** [comment]
@haroldrubio is it possible this must be implemented in the SAC console too? or only the PC console needs it?

**Review by melisabok — APPROVED** [review]

**Pull request merged**